monsieurkayode/More-Recipes-Kayode-Adeola
# #151257088 Setup Client-side application with React

#### What does this PR do?
* Have a client application to consume our API using React and Redux
* Have a React development environment to build out client-side application
* Configure all assets and packages
* Have reusable components for our pages
* Layout the view for the index page, dashboard page and recipe view page
#### Description of Task to be completed?
* Setup the React development environment
* Build React components
* Layout page views with static data
#### How should this be manually tested?
After cloning the repo, cd into it, run ` npm install` and then `npm run react`
You should see the following in the command console
```bash
> $ Starting development server
```
This will launch the application in the browser
#### Any background context you want to provide?
Certain features such as sidebar navigation and modal in mobile view do not work, for now, the CSS framework being used which is Materialize-CSS requires initialization using jQuery.  Still working on fixing this since it is bad practice using another library that modifies the DOM when using React
#### What are the relevant pivotal tracker stories?
#151257088
